shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/BridgeIngressServiceTest.java

```diff
@@ -63,11 +82,29 @@ public void testBridgeIngressCreation() {
     }
 
     @Test
-    @Disabled("Delete loop in BridgeIngressController does not get called. Bug in the SDK?")
     public void testBridgeIngressDeletion() {
         // Given
         BridgeDTO dto = new BridgeDTO(TestConstants.BRIDGE_ID, TestConstants.BRIDGE_NAME, "myEndpoint", TestConstants.CUSTOMER_ID, BridgeStatus.PROVISIONING);
 
+        // When
+        bridgeIngressService.createBridgeIngress(dto);
+        bridgeIngressService.deleteBridgeIngress(dto);
+
+        // Then
+        BridgeIngress bridgeIngress = kubernetesClient
+                .resources(BridgeIngress.class)
+                .inNamespace(customerNamespaceProvider.resolveName(dto.getCustomerId()))
+                .withName(KubernetesResourceUtil.sanitizeName(dto.getId()))
+                .get();
+        assertThat(bridgeIngress).isNull();
+    }
+
+    @Test
+    @Disabled("Delete loop in BridgeIngressController does not get called. Bug in the SDK?")
```

Comment:
Unrelated to this PR but do we have a ticket to track this problem?

Reply:
Just to clarify: as far as I've seen this is a problem of the testing framework, because this works on a real kubernetes cluster. I'm going to create a ticket for this test

Reply:
added link to ticket